e2e/helm_test.sh

```diff
@@ -40,6 +40,7 @@ set_variables() {
   HELM_CHART_NAME="awspca/aws-privateca-issuer"
   AWS_REGION="us-east-1"
   DEPLOYMENT_NAME="aws-privateca-issuer"
+  VALUES_FILE="test-values.yaml"
```

Comment:
So I tested this locally, and what happens is that the lines about mounting fail, because it uses the public helm chart and not the local one in the repo (i.e., the one with your changes). 

I was able to fix it by locally changing HELM_CHART_NAME to the relative path of the local helm chart, i.e.
reference the directory and then use the filenames from there. (This also made the script able to use test-values.yaml without being in the e2e directory). This isn't actually your error, but it does make the test fail without fixing it.

This is how my set_variables() function looks like, which makes the test work. Can you modify it to something like this?

```
set_variables() {
  DIR="$( dirname -- "$0"; )"
  K8S_NAMESPACE="aws-privateca-issuer" # also updated this for consistency
  HELM_CHART_NAME="$DIR/../charts/aws-pca-issuer"
  AWS_REGION="us-east-1"
  DEPLOYMENT_NAME="aws-privateca-issuer"
  VALUES_FILE="$DIR/test-values.yaml"
}
```

Reply:
I haven't been sure if you override this in your CI.   
The PR is updated with the change requests and is rebased.

Reply:
Thanks for the changes! Actually, I think currently this particular file is not part of the CI. Which is our miss, everything else is actually tested there.